### PR TITLE
Search city name

### PR DESCRIPTION
**Pull request by Kacique**

### CODE REVIEW

**Review by lindakw — APPROVED** [review]
Looks good!

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@ web-build/
 
 # macOS
 .DS_Store
+
+.private.js
```
Comment (Amele-A) [inline .gitignore:17]: I learn something new, thank you. This is a great way to use it.

**Review by Amele-A — APPROVED** [review]
RTK Query is a powerful data fetching and it's great using here on your code.  code looks great

**Comment by kmausolf** [comment]
Excellent work setting up RTK Query!

**Pull request merged**